src/main/java/psi/model/DualItem.java

```diff
@@ -0,0 +1,50 @@
+package psi.model;
+
+public class DualItem implements Item {
+
+	private String value;
+	private String option1;
+	private String option2;
+	private ContainerItem items;
+
+	public DualItem(String value, String option1, String option2, ContainerItem items) {
+		super();
+		this.value = value;
+		this.option1 = option1;
+		this.option2 = option2;
+		this.items = items;
+	}
+
+	public String getValue() {
```

Comment:
Not sure how setter / getter philosophy works in Java :) but all these setters and getters here seem to clutter a little the code.
* Are all of them needed? (e.g. the setters, since the values can also be injected via constructor args?)
* Are all of them used from outside? Could they be private?
* Is there any other more condensed notation? :D

Check this out:
https://softwareengineering.stackexchange.com/a/285193/154645

Reply:
"As stated in quite a few answers and comments, DTOs are appropriate and useful in some situations, especially in transferring data across boundaries (e.g. serializing to JSON to send through a web service)." -> this seems to be exactly my case. Maybe I could make the setters for some classes private, but for others an "update" behaviour is part of the requirement. Again, I'm not sure how well will the framework function without getters and setters. But I am fairly sure that serialization and deserialization to/from JSON will not work without getters and setters.

Reply:
Fair enough :). It's a pity there is no condensed version though 🤓